Disqus.php

```diff
@@ -270,12 +277,19 @@ protected function setOptions(array $options)
      */
     protected function httpRequest($url, $method = RequestInterface::METHOD_GET)
     {
-        $buzz = new Browser(new Curl());
+        $curl = new Curl();
+        $curl->setTimeout($this->timeout);
+        $buzz = new Browser($curl);
 
         try {
             $response = $buzz->call($url, $method);
         } catch (\RuntimeException $e) {
-            return false;
+            return json_encode(
+                array(
+                    'code' => $e->getCode(),
+                    'response' => $e->getMessage(),
+                )
+            );
```

Comment:
I think this will affect how "bad" requests are handled, as suddenly there *will* be a response key here: https://github.com/jlaso/KnpDisqusBundle/blob/master/Disqus.php#L118

So, I'm not sure we should do this.